arm-network/2016-12-01/swagger/serviceCommunity.json

```diff
@@ -0,0 +1,153 @@
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2016-12-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/bgpServiceCommunities": {
+      "get": {
+        "tags": [
+          "BgpServiceCommunities"
+        ],
+        "operationId": "BgpServiceCommunities_List",
+        "x-ms-examples": {
+          "ServiceCommunityList": { "$ref": "../examples/ServiceCommunityList.json" }
+        },
+        "description": "Gets all the available bgp service communities.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of BgpServiceCommunity resources.",
+            "schema": {
+              "$ref": "#/definitions/BgpServiceCommunityListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "BGPCommunity": {
+      "properties": {
+        "region": {
+          "type": "string",
+            "description": "The region which the service support. e.g. For O365, region is Global."
+        },
+        "communityName": {
+          "type": "string",
+          "description": "The name of the bgp community. e.g. Skype."
+        },
+        "communityValue": {
+          "type": "string",
+          "description": "The value of the bgp community. For more information: https://docs.microsoft.com/en-us/azure/expressroute/expressroute-routing."
+        },
+        "communityPrefixes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The prefixes that the bgp community contains."
+        }
+      },
+      "description": "Contains bgp community information offered in Service Community resources."
+    },
+    "BgpServiceCommunityPropertiesFormat": {
+      "properties": {
```

Comment:
looks like you missed defining a property named "state" over here. 
The example defines it as `state: "Enabled"` over [here](https://github.com/Azure/azure-rest-api-specs/pull/855/files#diff-b69d6fc6633988c248d41481e7d97081R16).